python/tvm/contrib/nvcc.py

```diff
@@ -98,7 +98,8 @@ def compile_cuda(code,
     (out, _) = proc.communicate()
 
     if proc.returncode != 0:
-        msg = "Compilation error:\n"
+        msg = code
```

Comment:
Is this intended change?

Reply:
Yes, I find it is useful to append code along with error messages during debugging